experiment.json

```diff
@@ -0,0 +1,39 @@
+{
+  "experiment": {"experimentId": "base_01"},
```

Comment:
If these files are supposed to be edited by humans, consider using TOML instead of JSON, a language specifically designed for configuration files used by Python and Rust:
https://github.com/toml-lang/toml#comparison-with-other-formats
https://www.python.org/dev/peps/pep-0518/#toml
https://doc.rust-lang.org/cargo/reference/manifest.html

Reply:
I added TOML as an option. The configuration file can now be in either .toml or .json format